.releaserc

```diff
@@ -0,0 +1,26 @@
+{
+  "ci": false,
+  "branches": ["main", {"name": "alpha", "prerelease": true}, {"name": "beta", "prerelease": true}],
+  "plugins": [
+    ["@semantic-release/commit-analyzer", {
+      "preset": "angular",
+      "releaseRules": [
+        {"type": "dep", "release": "patch"}
```

Comment:
Are these exclusive, or in addition to preset `angular`?

Reply:
in addition. so angular + whatever rules we decide.